src/modules/fw_rate_control/FixedwingRateControl.cpp

```diff
@@ -360,8 +360,10 @@ void FixedwingRateControl::Run()
 				const Vector3f angular_acceleration_setpoint = _rate_control.update(rates, body_rates_setpoint, angular_accel, dt,
 						_landed);
 
-				const float roll_feedforward = _param_fw_rr_ff.get() * _airspeed_scaling * body_rates_setpoint(0);
-				const float pitch_feedforward = _param_fw_pr_ff.get() * _airspeed_scaling * body_rates_setpoint(1);
+				const float roll_feedforward = _param_fw_rr_ff.get() * _airspeed_scaling * body_rates_setpoint(
+								       0) + _param_fw_ra_ff.get() * _rates_sp.accel_feedforward[0];
```

Comment:
just curious on the design choice here .. why put the accel feedforward in the rate setpoint? this requires an extra gain to tune, whereas you could just pass the accel feedforward directly through the rate controller, summing it with the feedback accel term